packages/server/test/unit/plugins/child/run_plugins_spec.js

```diff
@@ -15,6 +16,17 @@ const Fixtures = require(`${root}../../test/support/helpers/fixtures`)
 const colorCodeRe = /\[[0-9;]+m/gm
 const pathRe = /\/?([a-z0-9_-]+\/)*[a-z0-9_-]+\/([a-z_]+\.\w+)[:0-9]+/gmi
 
+const deferred = () => {
+  let reject
+  let resolve
+  const promise = new Promise((_resolve, _reject) => {
+    resolve = _resolve
+    reject = _reject
+  })
+
+  return { promise, resolve, reject }
+}
```

Comment:
could also use `Bluebird.defer`, it's deprecated but i won't tell :shushing_face: